themes/src/main/resources-community/theme/base/login/messages/messages_pt_BR.properties

```diff
@@ -65,73 +74,119 @@ region=Estado
 postal_code=CEP
 country=Pa\u00EDs
 emailVerified=E-mail verificado
-gssDelegationCredential=gss delega\u00E7\u00E3o credencial
+gssDelegationCredential=Delega\u00E7\u00E3o de Credenciais GSS
+
+profileScopeConsentText=Perfil de usu\u00E1rio
+emailScopeConsentText=Endere\u00E7o de e-mail
+addressScopeConsentText=Endere\u00E7o
+phoneScopeConsentText=N\u00FAmero de telefone
+offlineAccessScopeConsentText=Acesso Offline
+samlRoleListScopeConsentText=Minhas Fun\u00E7\u00F5es
+rolesScopeConsentText=Fun\u00E7\u00F5es do usu\u00E1rio
+
+restartLoginTooltip=Reiniciar o login
+
+loginTotpIntro=Voc\u00EA precisa configurar um gerador de senha de uso \u00FAnico para acessar esta conta
+loginTotpStep1=Instale um dos seguintes aplicativos no seu celular:
+loginTotpStep2=Abra o aplicativo e digitalize o c\u00F3digo de barras:
+loginTotpStep3=Digite o c\u00F3digo \u00FAnico fornecido pelo aplicativo e clique em Ok para concluir a configura\u00E7\u00E3o.
+loginTotpStep3DeviceName=Forne\u00E7a um nome de dispositivo para ajud\u00E1-lo a gerenciar seus dispositivos OTP.
+loginTotpManualStep2=Abra o aplicativo e digite a chave:
+loginTotpManualStep3=Use os seguintes valores de configura\u00E7\u00E3o se o aplicativo permitir defini-los:
+loginTotpUnableToScan=N\u00E3o foi poss\u00EDvel digitalizar?
+loginTotpScanBarcode=Digitalizar c\u00F3digo de barras?
```

Comment:
`Ler QRCode?`. My point is that I hear more `ler` than `digitalizar` with respect to qrcodes.

Reply:
`QRCode` really sounds better than `Código de barras`. thanks